command/vault/secrets_delete.go

```diff
@@ -0,0 +1,75 @@
+package vault
```

Comment:
this is a breaking, unrelated change to your PR title, please open a new PR for discussion on this